### ISSUE TITLE
Output Labels are not in one-hot encoding for classification

### ISSUE THREAD

**Issue by sumitsinha** [charlesq34/pointnet#131]
The output labels for each point are single numbers between 1-40. The last output layer has 40 output units. I wanted to know should the labels be converted to one hot encoding (vector of 40 values) before giving it to the for model training?

**Comment by sumitsinha** [comment]
I see we are using sparse_softmax_cross_entropy_with_logits in the loss function that takes care of one-hot encoding problem.
